instalacion-entorno/README.md

```diff
@@ -0,0 +1,117 @@
+# Configuración de entorno
+
+## 1- Instalar Node
+
+Para instalar **Node** tenemos que irnos a la [página oficial de Node](https://nodejs.org/en/) donde nos aparecerán dos versiones de **Node** para descargar. Instalaremos la recomendada, ay que hay ocasiones que las ultimas versiones no funcionan con todos los paquetes.
+
+![Portal Node](./images/node.JPG)
+
+Cuando se haya descargado el ejecutable lanzaremos la instalación, dejaremos todos los pasos que vienen por defecto.
+
+Una vez instalado **Node** para comprobar si se ha instalado correctamente, abriremos una coonsola de comandos y ejecutaremos
+
+```
+node -v 
+```
+
+Cuando ejecutemos el comando debería de reconocer nuestro comando y debería mostrarnos la version de node que tenemos.
+
+![Versión Node](./images/versionNode.JPG)
+
+Con esto habriíamos acabado la instalación de **Node**
+
+
+## 2- Instalación Yarn
+
+Por defecto **Node** viene ya con un manejador de paquetes que es **npm** pero nosotros vamos a instalar **yarn** que es otro manejador de paquetes.
+
+**Yarn** nos da un mayor performance, offline cache que nos permite instalar aquellos paquetes que ya hemos instalado alguna vez en el pc sin conexión y tiene un nuevo algoritmo de instalación que nos va hacer olvidar **"En mi máquina funciona"**.
```

Comment:
¿Qué te parece algo así? 
"...nos permite instalar en modo sin conexión (offline cache) aquellos paquetes que ya hemos instalado alguna vez en el pc. Además, tiene un nuevo algoritmo de in....."